Client.Core/Client.Core.csproj

```diff
@@ -38,5 +39,4 @@
       <PackageReference Include="RestSharp" Version="106.6.10" />
       <PackageReference Include="System.Net.Http" Version="4.3.4" />
     </ItemGroup>
-    
```

Comment:
Please add conditional dependency to `Microsoft.Bcl.AsyncInterfaces`:

```xml
<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' ">
        <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
</ItemGroup>
```

Reply:
> 
> Could you give me a permission to update your `GridProtectionAlliance:feat/AsyncEnumerable` branch? I will fix our CircleCI scripts to supports dual targeting.
> 
> Regards

Write permissions granted. Will attempt your other requests.